NHSN/src/test/java/com/nhsn/pages/ReviewPage.java

```diff
@@ -54,6 +59,36 @@ public void clickOnGenerateButton()
         System.out.println("Clicked on Generate Button");
     }
 
+    public void enterSubmitDateOnReviewPage(String date) throws InterruptedException {
+
+        WebElement submitDateTextbox = driver.findElement(By.xpath("//input[@name='dp']"));
+        char dateArray[] = date.toCharArray();
```

Comment:
I like your approach. But this  function is unique because this textbox is not accepting SendKeys method. I had to enter one by one letter to trigger the search API behind scene. Going forward I will create separate function in CommonFunctions class and use it.